### PR TITLE
Sample data code enhancement

### PR DESCRIPTION
**Pull request by chmst**
### Summary of Changes
Sample Data store three categories. 
The code was copy/pasted three times. This PR makes a loop for it. 

Additionally params for the blog "help" were made as integers.
The blog "help" gets the site title, such adding a h1 tag to the site.

### Testing Instructions
Install sample data - everything should work as before and store 3 categories with their articles. 
Check if the page help has a h1 tag.

### Note
This does not resolve the problem with adding persian language

### CODE REVIEW

File: plugins/sampledata/blog/blog.php [hunk 1]
```diff
@@ -9,6 +9,7 @@
 
 defined('_JEXEC') or die;
 
+use FG\ASN1\OID;
```
Comment (richard67) [inline plugins/sampledata/blog/blog.php]: What is this?

File: plugins/sampledata/blog/blog.php [hunk 2]
```diff
@@ -19,6 +20,7 @@
 use Joomla\CMS\Plugin\CMSPlugin;
 use Joomla\CMS\Session\Session;
 use Joomla\Database\ParameterType;
+use phpDocumentor\Reflection\Types\Object_;
```
Comment (richard67) [inline plugins/sampledata/blog/blog.php]: And this?

**Comment by chmst** [comment]
i don't know, it came after the last pull from 4.0.

**Comment by richard67** [comment]
@chmst It's definitely not in the file on the 4.0-dev branch of the CMS.

**Review by richard67 — APPROVED** [review]

**Comment by richard67** [comment]
I have tested this PR :white_check_mark: with success.

**Comment by richard67** [comment]
Thanks!

**Pull request merged**